core/model/objects/peer.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+http://soramitsu.co.jp
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef CORE_DOMAIN_OBJECTS_PEER_HPP_
+#define CORE_DOMAIN_OBJECTS_PEER_HPP_
+
+#include <string>
+#include <memory>
+
+namespace object {
+
+class Peer {
+
+public:
+    std::string     ip;
+    std::string     publicKey;
+
```

Comment:
For each peer `sumeragi.json` has:
 - name
 - ip
 - publicKey

Should `name` be here?